Sources/ResearchKitOnFHIR/FHIRToResearchKit/QuestionnaireItem+ResearchKit.swift

```diff
@@ -151,7 +151,11 @@ extension QuestionnaireItem {
             guard !answerOptions.isEmpty else {
                 throw FHIRToResearchKitConversionError.noOptions
             }
-            return ORKTextChoiceAnswerFormat(style: ORKChoiceAnswerStyle.singleChoice, textChoices: answerOptions)
+            var choiceAnswerStyle = ORKChoiceAnswerStyle.singleChoice
+            if itemControl == "multi-select" {
+                choiceAnswerStyle = .multipleChoice
```

Comment:
🎉🚀

Reply:
There is an inaccuracy here, the correct code to use in this case would be `check-box` as noted in the IG https://fhir-ru.github.io/codesystem-questionnaire-item-control.html. This has been updated in [d2abc38](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/64/commits/d2abc38fdf8f8ec741a72e8a70d71a62aea9ce1d).

Reply:
There is an error here, `multi-select` appears to not be a valid code per the SDC IG. It should be `check-box`, as shown [here](http://hl7.org/fhir/R4/valueset-questionnaire-item-control.html). I've updated the PR in [d2abc38](https://github.com/StanfordBDHG/ResearchKitOnFHIR/pull/64/commits/d2abc38fdf8f8ec741a72e8a70d71a62aea9ce1d)